Clarification about the Verifier in the Introduction (#46)
lamps-wg/csr-attestation: draft-ietf-lamps-csr-attestation.md

# Introduction

At the time that it is requesting a certificate from a Certification Authority (CA), a PKI end entity may wish to provide evidence of the security properties of the environment in which the private key is stored.

> This evidence is to be verified by a Verifier and the outcome of this evaluation is then made available to the Registration Authority (RA) or the Certification Authority (CA) as part of validating an incoming certificate request against a given certificate policy.

Reviewer: ```suggestion
This Evidence can be appraised by authoritative entities, such as a Registration Authority (RA) or a CA as part of validating an incoming certificate request against given certificate policies.
```

I'd prefer the phrasing that I proposed in #88, as we do not have to, for example, reintroduce `CA` that was just introduced in line 88 above and #88 also fixes inconsistent language, such as "evidence is to be verified".
